scripts/test

```diff
@@ -25,7 +25,7 @@ if [ "${BASH_SOURCE[0]}" = "${0}" ]; then
             *.py ./**/*.py \
             *.md
 
-        pytest --cov=stactools.ephemeral tests
+        pytest --cov=stactools.nisar_sim tests -s -rP
```

Comment:
Should we note somewhere the reason/meaning of the pytest flags you set?

Reply:
Meant to remove those - aren't necessary to run the tests in nisar_sim